internal/mode/static/nginx/config/telemetry_template.go

```diff
@@ -0,0 +1,22 @@
+package config
+
+const otelTemplateText = `
+otel_exporter {
+	endpoint "{{ .Endpoint }}";
```

Comment:
since now we have tight validation rules for endpoint, interval and service name that don't allow whitespace characters or anything like that, we don't really need to places those values in `"..."`. I suggest removing them to avoid any confusion in the future